compiler/modules/types/list.bal

```diff
@@ -289,6 +291,18 @@ function listInhabited(Context cx, FixedLengthArray members, SemType rest, Conju
     }
 }
 
+function listMemberInhabited(Context cx, Conjunction neg, FixedLengthArray members, SemType rest, ListAtomicType nt, int i) returns boolean {
```

Comment:
@jclark I'm having a hard time naming this function, this is just a temporary name.

Reply:
Not clear to me that we need this function.